app/assets/javascripts/charts.js

```diff
@@ -259,7 +260,12 @@ function get_pie_chart(div, url) {
       $.getJSON(url, function(data) {
         var target = $("#"+div+"-body");
         target.empty();
+        var hostsCount = 0;
+        $.each(data.values,function() {
+          hostsCount += this.data;
+        });
         expanded_pie(target, data.values);
+        $('.modal-title').empty().append('<b>' + data.name + ' </b>' +  __('Distribution Chart') + '<small> ('+ hostsCount + ' ' + __('hosts') +')</small>');
```

Comment:
use n_() for plural text ("hosts"), and don't concatenate numbers+words outside of a translated string